shared/lib/test_server.py

```diff
@@ -31,24 +33,31 @@ def find_open_port():
         return port
 
 
-# Start NL server on an unused port, so multiple integration tests can
-# run at the same time.
-if os.environ.get('DEFAULT_NL_SERVER'):
-  port = 6060
-else:
-  port = find_open_port()
+is_nl_mode = os.environ.get('ENABLE_MODEL') == 'true'
+if is_nl_mode:
+  # Start NL server on an unused port, so multiple integration tests can
+  # run at the same time.
+  if platform.system() == 'Darwin' and platform.processor() == 'arm':
+    msg = '\n\nIMPORTANT NOTE: Detected MacOS ARM processor! You need ' \
```

Comment:
nice! Can we make this msg more prominent with a lot of "===" or "!!!"

Reply:
Done.